src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -11,67 +11,100 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
+import aiNewRelicAiExplainThisError from 'images/ai_screenshot-crop_new-relic-ai-explain-this-error.webp'
 
-<Video
-  type="youtube"
-  id="XMSUVmNKVyM"
-/>
+import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-error-pop-up.webp'
+
+import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
+
+New Relic AI (NRAI) is designed to help every member of your team improve observability, not just engineers. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
 
 We help you get to the heart of your system by offering two quick and easy ways to ask questions: AI chat, and the NRQL console. 
 
-### Chat with AI [#ask-grok]
+### Ask AI questions [#ask-AI]
 
-    <img
-        title="Screenshot showing the ask grok button in the NRQL console"
-        alt="ask grok in the nrql console"
-        src={nrqlconsole}
-    />    
+You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages.
 
-You can work with our AI any time you want by using our interactive chat experience. All you have to do is click the **Ask AI** button available on the top right of most New Relic platform pages. 
+New Relic AI provides a variety of capabilities to help you:
+* Transform natural language into queries (NRQL/Nerdgraph)
+* Detect anomalies
+* Analyze error logs
+* Understand stack traces
+* Leverage our docs to understand New Relic platform concepts
 
 Our AI has been trained to communicate with engineers using common questions and plain language. Let's say you run an online store, and you start receiving alerts that response times for your `Checkout-service` are lagging. If you don't have time to dig into the issue, you can just ask New Relic AI "What's on fire?" or "What should I handle first today?" New Relic AI can tell you what's broken in your system and help you explore data with charts and tables.
 
-### NRQL console [#NRQL-console]
+### Get help creating NRQL queries [#NRQL-console]
+
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
 
 <img
```

Comment:
I'm not seeing an NRAI button in this screen shot. Also, you'll want to remove "Grok" references in the image properties.

Reply:
It's part of the NRQL console but I can highlight it in green!

Reply:
Done